### PR TITLE
12.0 fds modules migration

### PR DESCRIPTION
**Pull request by quentingigon**
This PR contains the 3 following FDS modules:
- l10n_ch_fds_postfinance
- l10n_ch_fds_upload_dd
- l10n_ch_fds_import_bank_statement

### CODE REVIEW

**Comment by ecino** [comment]
I created a branch for FDS migration. Please rebase to keep only FDS commits.

**Comment by quentingigon** [comment]
Need this PR first -> [HERE](https://github.com/CompassionCH/account-payment/pull/2)

**Comment by quentingigon** [comment]
The tests need _l10n_ch_pain_direct_debit_ to pass but it is not present on this branch

**Comment by quentingigon** [comment]
this error:
l10n_ch_payment_return_sepa/wizards/payment_return_import.py:8: [W0622(redefined-builtin), ] Redefining built-in 'UserWarning'
is fixed in PR for **l10n_ch_payment_return_sepa**

**Review by ecino — CHANGES_REQUESTED** [review]
Please remove module l10n_ch_fds_upload_sepa from this pull request (along with module l10n_ch_payment_return_sepa) and move it a new PR on this branch: https://github.com/CompassionCH/l10n-switzerland/tree/12.0-fds-sepa

File: l10n_ch_pain_direct_debit/demo/dd_demo.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ecino) [inline l10n_ch_pain_direct_debit/demo/dd_demo.xml]: This file should not be included in this PR, it was already done in branch https://github.com/CompassionCH/l10n-switzerland/tree/12.0-pain-direct-debit

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+git+https://github.com/OCA/openupgradelib
```
Comment (ecino) [inline requirements.txt]: This is not needed (was for pain_direct_debit)

**Review by ecino — CHANGES_REQUESTED** [review]

File: l10n_ch_fds_postfinance/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    'data': [
+        'data/import_files_cron.xml',
+        'views/fds_authentication_keys_view.xml',
+        'views/fds_files_import_from_fds_wizard_view.xml',
+        'views/fds_key_clone_wizard_view.xml',
+        'views/fds_key_generator_wizard_view.xml',
+        'views/fds_key_import_wizard_view.xml',
+        'views/fds_postfinance_account_view.xml',
+        'views/fds_postfinance_file_view.xml',
+        'security/ir.model.access.csv',
```
Comment (ecino) [inline l10n_ch_fds_postfinance/__manifest__.py:42]: This file is missing! Did you even try to install the module?

**Review by ecino — CHANGES_REQUESTED** [review]

File: l10n_ch_fds_postfinance/__manifest__.py (hunk 3 above)
Comment (quentingigon) [reply l10n_ch_fds_postfinance/__manifest__.py:42]: J'ai du faire une fausse manip et le supprimer car je pouvais l'installer avant.. je vais checker ça

**Review by ecino — CHANGES_REQUESTED** [review]
Missing two modules that you must migrate as well from v11:
- `l10n_ch_fds_import_bank_statement`
- `l10n_ch_fds_import_payment_return`

These can be found on this branch https://github.com/CompassionCH/l10n-switzerland/tree/11.0-everything-migrated

Did you as well take this version for other modules fds ? Please make sure that the version corresponds.

**Comment by ecino** [comment]
Please rebase your branch and:
- Drop commit https://github.com/CompassionCH/l10n-switzerland/pull/43/commits/702d49c6f5a0f793be7fc83ccfbe275ffb57eefb
- Keep one commit for each module migration
- Keep maybe one commit for the fixes if needed (if you cannot merge with other commits)
- Don't forget to put CO-3126 in your commit messages

**Comment by ecino** [comment]
Manually merged.
